### ISSUE TITLE
add TP and SL to tradingview alerts

### ISSUE THREAD

**Issue by blortmort** [junta/tradingview-alert-connector#29]
its too risky to depend on the trading bot to close trades when trading large amounts, I suggest you add a way to add a TP and SL through alerts. Instead of having no stop loss, its very dangerous

**Comment by Seblulu** [comment]
DO you know how to add a TP or SL to it ? Please

**Comment by junta** [comment]
yes, I agree with you that it's risky, especially trading large amounts. 
Supporting TP and SL is in my mind, but not implemented yet.

**Comment by Seblulu** [comment]
Can we use the TP and SL from the strategy itself ?

**Comment by junta** [comment]
yes, when TP or SL is triggered on tradingview side, this connector tool receive the alert and send corresponding order

For TP or SL on Tradingview:
https://www.tradingview.com/pine-script-docs/en/v5/concepts/Strategies.html?highlight=take%20profit

**Comment by Seblulu** [comment]
Thanks a lot
